software/inc/encoding.h

```diff
@@ -1455,6 +1456,7 @@ DECLARE_CSR(mhpmcounter28h, CSR_MHPMCOUNTER28H)
 DECLARE_CSR(mhpmcounter29h, CSR_MHPMCOUNTER29H)
 DECLARE_CSR(mhpmcounter30h, CSR_MHPMCOUNTER30H)
 DECLARE_CSR(mhpmcounter31h, CSR_MHPMCOUNTER31H)
+DECLARE_CSR(mcountinhibit,CSR_MCOUNTINHIBIT)
```

Comment:
What's the purpose of this new CSR?

Reply:
it's a standard RISC-V register, which is now in the CPU and that we use. it's used to enable performance counters